src/demo/ui.jsx

```diff
@@ -46,17 +39,14 @@ const styles = {
     backgroundColor: colors.white,
     fontSize: '120%',
     borderRadius: 8,
-    border: `2px solid ${colors.black}`,
     minWidth: 160,
     padding: '16px 30px',
-    ':focus': {
```

Comment:
this didn't work when i added it in #130 😢but should work once Radium is actually being used

Reply:
Doesn't this PR start it being used?

Reply:
i installed Radium, but haven't actually set it up yet. i'll do that in a follow-up PR to keep this one small-ish